grants/search.js

```diff
@@ -335,8 +329,8 @@ async function buildMatchCriteria(queryParams) {
     /**
      * Search queries
      *
-     * We want search terms to be treated as logical AND querries.
-     * By default mongodb peforms as logical OR search of the terms unless specified as a phrase.
+     * We want search terms to be treated as logical AND queries.
+     * By default mongodb performs as logical OR search of the terms unless specified as a phrase.
```

Comment:
🙈 Gotta get me a spellchecker for my editor.

Reply:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker